docs/source/reference/api-server/examples/api-server-auth-proxy.rst

```diff
@@ -136,3 +136,5 @@ During the login flow, the token provided by the web login will encode the cooki
 .. note::
 
     If your auth proxy is not automatically detected, try using ``sky api login --cookies`` to force auth proxy mode.
+
+If the `X-Auth-Request-Email` header is set by your auth proxy, SkyPilot will use it as the user name all requests.
```

Comment:
```suggestion
If the `X-Auth-Request-Email` header is set by your auth proxy, SkyPilot will use it as the username in all requests (e.g., `alice@skypilot.co` will be displayed in UI and CLI).
```

Reply:
How about moving this paragraph to end of last section? This sec is optional.

Can also add a screenshot of the UI showing `ldap@sky.your.com`.

Reply:
<img width="589" alt="Screenshot 2025-05-22 at 19 53 29" src="https://github.com/user-attachments/assets/b61e37c0-7cf6-426d-93b3-e5c25f67cff0" />

Reply:
This paragraph is meant to tell the admin which header to set, if they are using a custom auth proxy.

I will add some detail about the UX in the "User flow" section.